### PR TITLE
Governance pre-release final-ish touches

### PR DESCRIPTION
**Pull request by emccorson**
### Changed
- Temporarily switch off proposal persistent cache
- Temporarily hide placeholder vote info text
- Default to highest vote in proposal summary

### Added
- Add success/failure notifications for voting

### Removed
- Remove ProposalAndVote error handling code (can be done better elsewhere)

### Fixed
- Readd proposals table filter and search
- Use indexer to check if user voted on proposal
- Make the vote button read "Edit Vote" once the user has voted
- Fix some text in notifications
- Add footer to proposal page
- Fix panel gaps on proposal page

### CODE REVIEW

File: apps/namadillo/src/slices/proposals/functions.ts [hunk 1]
```diff
@@ -308,71 +351,33 @@ export const fetchVotedProposalIds = async (
...
+): Promise<TransactionPair<VoteProposalProps>[]> => {
   try {
-    const signedTx = await signingClient.sign(
-      signer,
-      txArray.map(({ tx }) => tx)
+    const { tx } = await getSdkInstance();
+
+    const voteProposalProps = {
+      signer: account.address,
+      proposalId: BigInt(1),
```
Comment (pedrorezende) [inline apps/namadillo/src/slices/proposals/functions.ts:366]: Not sure if this is intended, because we have `proposalId` param unused

**Review by pedrorezende — APPROVED** [review]
Looks really good :) Before merging, could you please just take a look at the comment I made to check if something was left behind?

File: apps/namadillo/src/slices/proposals/functions.ts (hunk 1 above)
Comment (emccorson) [reply apps/namadillo/src/slices/proposals/functions.ts]: Oh nice spot! That is definitely a mistake. I've fixed it and added a commit which will make the linter catch this error.

**Review by mateuszjasiuk — APPROVED** [review]
LGTM!

**Pull request merged**